src/main/java/uk/gov/justice/digital/delius/controller/secure/CustodyResource.java

```diff
@@ -122,4 +123,16 @@ public Custody getCustodyByConvictionId(final @PathVariable String crn,
         return custodyService.getCustodyByConvictionId(crn, convictionId);
     }
 
+    @RequestMapping(value = "offenders/nomsNumber/{nomsNumber}/recall", method = RequestMethod.PUT, consumes = "application/json")
```

Comment:
very picky, but I think it should be `/recalled`   since essentially the recall has happened  rather than this is requesting a recall

Reply:
np - will update